### PR TITLE
Toggle Mobile Menu

### PR DESCRIPTION
**Pull request by jaredatandi**
## In the mobile menu, I implemented the following features:

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed.

### CODE REVIEW

**Review by ha-manel — COMMENTED** [review]
# STATUS: INVALID 🔴 
Hi Team
I'm sure you put a lot of effort into this project but this PR is invalid because you didn't setup `eslint` for this project 😓 Since you're using JavaScript, you have to setup Eslint too 👍 

Kindly fix the issue and submit the PR for review again 👍 
Best of luck!

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jaredatandi** [comment]
Hi @ha-manel kindly could you be more detailed in the comment because I have all the necessary files setup as you can see from the screenshot, I do not understand your request fully 
image

**Comment by jaredatandi** [comment]
Thank you for taking the time to review my code, I have handled the requested changes.

**Comment by ha-manel** [comment]
I was about to respond to your previous comment 😅 I'm glad you figured it out, good job 👏

File: css/style.css [hunk 1]
```diff
@@ -176,144 +180,182 @@ body {
...
-  .career {
-    color: var(--green);
-  }
+.ham-menu {
+  padding-right: 1rem;
+  padding-left: 320px;
+  display: flex;
+  align-items: flex-start;
+  position: absolute;
+}
```
Comment (GzimAsani) [inline css/style.css]: - [ ] The hamburger button and X should be removed on desktop, kindly update it with usage of media queries 
image

File: css/style.css [hunk 2]
```diff
@@ -176,144 +180,182 @@ body {
...
 
-  .menu {
+  .x-icon {
     display: none;
   }
 
-  /* container */
-  .container {
+  .menu {
     display: flex;
```
Comment (GzimAsani) [inline css/style.css:199]: - [x] You have used `display: flex` a lot of times, which falls under the category of repeating yourself which is bad practice. So I would suggest you create a class  like `d-flex` and use it whenever you need it on your HTML

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello  @jaredatandi                :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with styling looks like the design :+1: 

 ## Required Changes 


- [x] Check the comments below :arrow_double_down: 







 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 2 above)
Comment (jaredatandi) [reply css/style.css:199]: :laughing: kindly sir, let me handle code refactoring when my head is not on fire. It is best, for now, to focus on this activity's requirements.

File: css/style.css (hunk 1 above)
Comment (jaredatandi) [reply css/style.css]: Thank you for taking the time to review my code :+1: 
I have implemented this change.

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅
## Hi Team👋

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**



## Highlights:
* Well written PR description ✔️ 
* All tests passed ✔️ 


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**